client/components/meta-inputs/meta-datetime/Edit/index.vue

```diff
@@ -1,10 +1,11 @@
 <template>
   <div class="d-flex">
     <date-picker
-      @change="setDate"
+      @input="setDate"
       :value="date"
       :label="meta.label"
-      :placeholder="meta.placeholder" />
+      :placeholder="meta.placeholder"
+      :clearable="false" />
```

Comment:
@underscope AFAIU it shouldn't since this component has its own `clear` button rendered outside of the date picker.
I tested the component by adding `date` meta and it seems to behave the same.